huaweicloud/services/elb/resource_huaweicloud_elb_l7policy.go

```diff
@@ -73,12 +90,21 @@ func resourceL7PolicyV3Create(ctx context.Context, d *schema.ResourceData, meta
 		return diag.Errorf("error creating ELB client: %s", err)
 	}
 
+	action := "REDIRECT_TO_POOL"
```

Comment:
how about move this default value to schema

Reply:
done